src/backend/d3d12/CommandBufferD3D12.cpp

```diff
@@ -256,6 +256,36 @@ namespace d3d12 {
                         D3D12_RECT scissorRect = { 0, 0, static_cast<long>(width), static_cast<long>(height) };
                         commandList->RSSetViewports(1, &viewport);
                         commandList->RSSetScissorRects(1, &scissorRect);
+
+                        for (uint32_t slot = 0; slot < currentRenderPass->GetAttachmentCount(); ++slot) {
```

Comment:
+1 on doing it in BeginSubpass because it should be valid to use the attachments as textures in previous subpasses.